packages/marko-web-theme-monorail/components/nodes/section-feed-content.marko

```diff
@@ -146,9 +146,19 @@ $ const blockName = "section-feed-content-node";
         />
       </else-if>
 
-      <marko-web-content-sponsors|{ node }| block-name=blockName obj=content>
-        Sponsored By: <marko-web-content-name tag="span" block-name=blockName obj=node link=true />
-      </marko-web-content-sponsors>
+      <if(content.sponsors && getAsArray(content, "sponsors.edges").length)>
+        $ const sponsorEdges = getAsArray(content, "sponsors.edges");
+        <marko-web-element block-name=blockName name="content-sponsors">
+          Sponsored By:
+          <for|sponsor, index| of=sponsorEdges>
+            $ const sponsorNode = getAsObject(sponsor, "node");
+            <span>
+              <marko-web-content-name tag="span" block-name=blockName obj=sponsorNode link=true />
+              <if(index + 1 !== sponsorEdges.length)>, </if>
+            </span>
+          </for>
+        </marko-web-element>
+      </if>
```

Comment:
@Shinsina If we want to use the same internal components as before it would look something like 
```markojs
<marko-web-element block-name=blockName name="content-sponsors">
  <marko-web-obj-nodes|{ node, index, length }| type="content" block-name=blockName obj=content field="sponsors" tag="span">
    <if(index === 0)>
      <marko-web-element tag="span" block-name=blockName name="content-sponsors-label">Sponsored By: </marko-web-element>
    </if>
    <marko-web-content-name tag="span" block-name=blockName obj=node link=true />
    <if(index + 1 !== length)>, </if>
  </marko-web-obj-nodes>
</marko-web-element>
```